app/src/main/java/io/mochahub/powermeter/exercises/ExerciseAdapter.kt

```diff
@@ -0,0 +1,36 @@
+package io.mochahub.powermeter.exercises
```

Comment:
Moved exercise screen related code to an `exercises` package which I think we should do for each screen